book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,141 @@
+
+
+(er-activism-cultural-change-emergent-strategy)=
+## Emergent strategy
+
+Emergent strategy by [adrienne marie brown](https://adriennemareebrown.net/) ({cite:ps}`brown2017emergent`) teaches us that change is inevitable, and that we can shape change. 
+By practising at the small scale/fractals we set the patterns for the whole system. 
+We all hold pieces of the solution and will need to transform ourselves to transform the world. 
+By being accountable, practising generosity and vulnerability, we can connect better to others build lasting relationships. 
+The easier it is for you to be wrong, the faster you can release your viewpoint and the easier it is to adapt to changing circumstances. 
+
+**Principles of emergent strategy**
+*	Small is good, small is all (the large is a reflection of the small).
+*	Change is constant (be like water).
+*	There is always enough time for the right work. 
+*	There is a conversation in the room that only this moment can have. Find it.
+*	Never a failure, always a lesson.
+*	Trust the people. (If you trust the people they become trustworthy or the boundaries will become clear.)
+*	Move at the speed of trust. Focus on critical connections more than critical mass – build the resilience by building the relationships.
+*	Less prep, more presence.
+*	What you pay attention to grows.
+
+Conversation is a crucial way to explore what we believe and to make new understandings and ideas possible. 
+Consider the areas where trust needs to be built: you will need to demonstrate that you're thinking about the same values/concerns. 
+To accomplish your vision for the future it needs to be clear what needs to be done and who will do it. 
+There should be time for reflection and evaluation to apply lessons learned to future work, and time for celebration of collective achievements. 
+
+(er-activism-cultural-change-transitions)=
+## Managing Transitions
+
+Managing Transitions by William and Susan Bridges ({cite:ps}`Bridges2017transitions`) teaches us that a change will always be accompanied by a transition. 
+A transition begins when something needs to be let go, and sits in between the old and the new situation. 
+This transition phase can be insecure for people who will have to adjust to a new situation, but also an opportunity for innovation to happen if people are adequately resourced and supported (by policies, groupings, procedures). 
+
+There are four P's in transitions ({cite:ps}`Bridges2017transitions`): 
+- Purpose: people have to understand the logic behind the outcome and feel that the change is necessary.
+- Picture: people need to experience how the outcome will look and feel.
+- Plan: people need a step-by-step plan to know what they have to do.
+- Part to play: especially if people have something to loose, they need to be able to play a part in the transition phase and eventual outcome.
+
+To manage transitions it needs to be clear what the problem (reason for change) is. 
```

Comment:
```suggestion
To manage transitionsm, it needs to be clear what the problem (reason for change) is. 
```

Reply:
Focus of the book is transitions, so I think we should keep the same phrasing here.